platform/ios/src/MGLMapView+OpenGL.mm

```diff
@@ -168,6 +148,7 @@ void bind() override {
 }
 
 #ifdef MGL_RECREATE_GL_IN_AN_EMERGENCY
+// TODO: Fix or remove
 // See https://github.com/mapbox/mapbox-gl-native/issues/14232
 void MGLMapViewOpenGLImpl::emergencyRecreateGL() {
```

Comment:
I'm guessing we should never be getting into this situation where we need to recreate the glView anymore?

Reply:
🤞 
I did test this function, but it's currently broken, and it was never a good option anyway, and an even worse one for this particular issue. Let's remove in a future PR.